packages/token-vault/src/lib/client.ts

```diff
@@ -113,7 +158,12 @@ export function client(config: ClientConfig) {
             };
           });
         },
-        refresh() {
+
+        /**
+         * @method refresh - Refresh tokens in the Token Vault
+         * @returns {Promise<{ refreshTokens: boolean }>}
+         */
+        refresh(): Promise<{ refreshTokens: boolean }> {
```

Comment:
This allows the main app to directly ask the Proxy to refresh the Access Token. It does not return the tokens.